pkg/web/static/js/utils.js

```diff
@@ -0,0 +1,11 @@
+export function setSuccessToast(msg) {
```

Comment:
A follow on story will be created to use this function in other `js` files where a success toast message should be displayed.